packages/core/src/services/projects/import.ts

```diff
@@ -0,0 +1,76 @@
+import { env } from '@latitude-data/env'
+import { eq } from 'drizzle-orm'
+
+import { Project, User, Workspace } from '../../browser'
+import { database } from '../../client'
+import { Result, Transaction } from '../../lib'
+import { DocumentVersionsRepository } from '../../repositories'
+import { projects } from '../../schema'
+import { mergeCommit } from '../commits'
+import { createCommit } from '../commits/create'
+import { createNewDocument } from '../documents'
+import { createProject } from './create'
+
+export async function importDefaultProject(
+  {
+    workspace,
+    user,
+  }: {
+    workspace: Workspace
+    user: User
+  },
+  db = database,
+) {
+  if (!('DEFAULT_PROJECT_ID' in env) || env.DEFAULT_PROJECT_ID === undefined) {
```

Comment:
I mean just make the env var mandatory please 😂 all this null checking is unnecessary

Reply:
wait no actually it's fine that is optional since the OSS can't depend on our db state